setup.py

```diff
@@ -50,7 +50,11 @@
     ],
     python_requires='>=3.6',
     install_requires=[
-        "pyyaml",
         "jinja2",
+        "keyring",
```

Comment:
`keyring` actually doesn't appear to be required as it's not used anywhere.  I think it should also be dropped from `requirements.txt`.